src/Essentials/src/Share/Share.shared.cs

```diff
@@ -180,7 +180,7 @@ public ShareMultipleFilesRequest()
 		{
 		}
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/ShareMultipleFilesRequest.xml" path="//Member[@MemberName='.ctor'][1]/Docs" />
+		/// <include file="../../docs/Microsoft.Maui.Essentials/ShareMultipleFilesRequest.xml" path="//Member[@MemberName='.ctor'][3]/Docs" />
```

Comment:
The various scripts we previously used to import existing docs often got things mixed up with overloads. This fixes such mix-ups by selecting the correct overload.